Week 1/Day 3/recipes/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
```

Comment:
You're putting everything inside of `<div>` tags.  a lot of people don't instinctively do this when they start out.  👍